paper/user_study.tex

```diff
@@ -38,11 +38,11 @@ \subsection{Methodology}\label{subsec:methodology} % Minye
 \begin{itemize}
     \item Setup: Open the application and open schema and \cfgfile.
     \item Information Retrieval Questions: with increasing difficulty, the participant has to retrieve information from the \cfgfile{}.
-    Much easier to solve by using the GUI panel.
+    Easier to solve by using the GUI panel.
     \item Configuration Modifications: Different tasks that involve changing the \cfgfile{}.
     Intend is that the participant becomes more familiar with the GUI panel.
     \item Schema modifications: Making adjustments to the schema.
-    Most difficult, but still feasible using the GUI panel.
+    Most difficult tasks. Feasible using the GUI panel.
```

Comment:
Well, actually, it is just as feasible with the code editor, if you know json schema I guess

Reply:
@PaulBredl adjusted in the paper now